utils/tf_solver.py

```diff
@@ -94,7 +97,10 @@ def set_objective(self, model_init_fn, dataset, normalization):
                 lambda x, y: (data_aug_layer(x[None], training=True)[0], y),
                 num_parallel_calls=tf.data.experimental.AUTOTUNE,
             )
-        self.dataset = self.dataset.batch(
+        self.dataset = self.dataset.shuffle(
+            buffer_size=1000,  # For now a hardcoded value
```

Comment:
can't this be auto tuned?
if not, seems fine like this 😅

Reply:
Not possible unfortunately haha !